cms/djangoapps/contentstore/views/item.py

```diff
@@ -128,6 +132,9 @@ def xblock_handler(request, usage_key_string):
               these fields:
                 :parent_locator: parent for new xblock, required for both duplicate and create new instance
                 :duplicate_source_locator: if present, use this as the source for creating a duplicate copy
+                :move_source_locator: if present, use this as the source item for moving
```

Comment:
Why does this need to specify the source? Doesn't the usage key specify the block to be moved?